css/style.css

```diff
@@ -1,4 +1,4 @@
-#img-div {
+#img-divv {
   display: grid;
   place-items: center;
 }
```

Comment:
- [ ] This is an inline comment.

Reply:
@Olamarx 
i made use of an inline comment, but i don't know why it, didn't come up on this page, i made three inline comments